build.gradle.kts

```diff
@@ -17,7 +17,7 @@ plugins {
 }
 
 group = "io.modelcontextprotocol"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Could you please revert the version change? We will do this manually before the release.